### PR TITLE
Apptainer Official PPA for Ubuntu announcement

### PR DESCRIPTION
**Pull request by panda1100**
Signed-off-by: Yoshiaki Senda <yoshiaki@live.it>

### CODE REVIEW

File: src/posts/official-ppa-20230206.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+slug: "/news/official-ppa-20230206"
+date: "2023-02-06"
+title: "Announcing Apptainer Official PPA for Ubuntu"
+---
+
+We are pleased and excited to announce that Apptainer official PPA for Ubuntu 
+is now available!
+
+Rocky Linux users have long benefited from the availability of Apptainer on
```
Comment (DrDaveD) [inline src/posts/official-ppa-20230206.md:10]: For the apptainer announcement let's not bias it to Rocky.  Say "Users of RHEL-derived operating systems".

File: src/posts/official-ppa-20230206.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+---
+slug: "/news/official-ppa-20230206"
+date: "2023-02-06"
+title: "Announcing Apptainer Official PPA for Ubuntu"
+---
+
+We are pleased and excited to announce that Apptainer official PPA for Ubuntu
```
Comment (DrDaveD) [inline src/posts/official-ppa-20230206.md:7]: I think it's going a bit far to say excited for this announcement. I'd like you to remove "and excited" and change the exclamation point to a period.

File: src/posts/official-ppa-20230206.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+We are pleased and excited to announce that Apptainer official PPA for Ubuntu 
+is now available!
+
+Rocky Linux users have long benefited from the availability of Apptainer on
+the EPEL repository. Apptainer, formerly known as Singularity, is widely used
+in the HPC community. Ubuntu is the one another major distribution for
+Apptainer users. Now there is great news for Ubuntu and Windows/WSL2/Ubuntu
+Apptainer users! Apptainer pre-built binaries are now available on the PPA
+for Ubuntu repository. Now you can install Apptainer using apt package
+manager. This makes it easy to install and update Apptainer on Ubuntu and
```
Comment (DrDaveD) [inline src/posts/official-ppa-20230206.md:16]: Please remove the 3 sentences starting with "Apptainer," and going through the exclamation point.

Replace the next 2 sentences with this one: "Apptainer pre-built binaries are now also available in an Ubuntu PPA repository, installable with the apt package manager."

File: src/posts/official-ppa-20230206.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+sudo dnf install -y epel-release
+sudo dnf install -y apptainer-suid
+```
+
+### Update Setuid Installation
+
+```
+sudo dnf check-update
+sudo dnf update -y apptainer-suid
+```
```
Comment (DrDaveD) [inline src/posts/official-ppa-20230206.md:85]: Please remove the section about installing on Rocky Linux, that's not new.

File: src/posts/official-ppa-20230206.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+sudo add-apt-repository -y ppa:apptainer/ppa
+sudo apt install -y apptainer-suid
+```
+
+### Update Setuid Installation
+
+```
+sudo apt update
+sudo apt install --only-upgrade -y apptainer-suid
+```
```
Comment (DrDaveD) [inline src/posts/official-ppa-20230206.md:51]: This whole section can be replaced with
```
See the [Apptainer documentation](https://apptainer.org/docs/admin/latest/installation.html#install-ubuntu-packages) for installation instructions.
```

**Review by DrDaveD — CHANGES_REQUESTED** [review]

File: src/posts/official-ppa-20230206.md (hunk 2 above)
Comment (panda1100) [reply src/posts/official-ppa-20230206.md:7]: Thank you, I'm preparing to update PR right now.

File: src/posts/official-ppa-20230206.md (hunk 1 above)
Comment (panda1100) [reply src/posts/official-ppa-20230206.md:10]: Thank you, I'm preparing to update PR right now.

File: src/posts/official-ppa-20230206.md (hunk 3 above)
Comment (panda1100) [reply src/posts/official-ppa-20230206.md:16]: Thank you, I'm preparing to update PR right now.

File: src/posts/official-ppa-20230206.md (hunk 5 above)
Comment (panda1100) [reply src/posts/official-ppa-20230206.md:51]: Thank you, I'm preparing to update PR right now.

File: src/posts/official-ppa-20230206.md (hunk 4 above)
Comment (panda1100) [reply src/posts/official-ppa-20230206.md:85]: Thank you, I'm preparing to update PR right now.

**Review by DrDaveD — APPROVED** [review]

**Pull request merged**